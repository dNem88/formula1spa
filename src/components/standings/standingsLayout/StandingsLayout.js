import React from 'react'
import styles from './standingsLayout.module.css'
import {Outlet} from 'react-router-dom'
import CommonHeader from '../../common/titleHeader/CommonHeader'
import SelectForm from '../../register/Select/SelectForm'
const headerProps = {
    header: 'F1 Standings 2022',
    description: 'Live standings for the current F1 Season'
}

function StandingsLayout() {

    return (
        <section className={styles['wrapper']}>
            <CommonHeader {...headerProps}/>
            <SelectForm location={'standings'}/>
            <div className={styles['outlet-container']}>
                <Outlet/>
            </div>
        </section>
    )
}

export default StandingsLayout