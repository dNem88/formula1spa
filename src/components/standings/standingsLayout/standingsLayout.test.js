import React from 'react'
import renderer from 'react-test-renderer'
import StandingsLayout from './StandingsLayout'


jest.mock('../../register/Select/SelectForm', () => 'SelectForm')
jest.mock('../../common/titleHeader/CommonHeader', () => 'CommonHeader')

describe('Test ScheduleLayout',() => {
    it('Match Snapshot', () => {
        const snapshot = renderer.create(
            <StandingsLayout />
        ).toJSON()
        expect(snapshot).toMatchSnapshot()
        expect(snapshot.props.className).toBe('wrapper')
        expect(snapshot.children.length).toBe(3)
    })
   
})