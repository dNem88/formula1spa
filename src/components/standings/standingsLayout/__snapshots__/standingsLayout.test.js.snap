// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ScheduleLayout Match Snapshot 1`] = `
<section
  className="wrapper"
>
  <CommonHeader
    description="Live standings for the current F1 Season"
    header="F1 Standings 2022"
  />
  <SelectForm
    location="standings"
  />
  <div
    className="outlet-container"
  />
</section>
`;
